agency/README.md

```diff
@@ -8,17 +8,26 @@ This specification contains a collection of RESTful APIs used to specify the dig
 
 ## Table of Contents
 
-* [General Information](#general-information)
-  * [Versioning](#versioning)
-  * [Responses and Error Messages](#responses-and-error-messages)
-  * [Authorization](#authorization)
-* [Vehicles](#vehicles)
-* [Vehicle - Register](#vehicle---register)
-* [Vehicle - Update](#vehicle---update)
-* [Vehicle - Events](#vehicle---event)
-* [Vehicle - Telemetry](#vehicle---telemetry)
-* [Telemetry Data](#telemetry-data)
-* [Stops](#stops)
+- [Mobility Data Specification: **Agency**](#mobility-data-specification-agency)
+  - [Table of Contents](#table-of-contents)
+  - [General information](#general-information)
+    - [Versioning](#versioning)
+    - [Responses and Error Messages](#responses-and-error-messages)
+    - [Authorization](#authorization)
+  - [Vehicles](#vehicles)
+  - [Vehicle - Register](#vehicle---register)
+  - [Vehicle - Update](#vehicle---update)
+  - [Vehicle - Event](#vehicle---event)
+    - [Trip_id Requirements](#trip_id-requirements)
```

Comment:
Should this be a section on this page? This link and the 2 under it don't go anywhere.